docs/manual/manual.adoc

```diff
@@ -233,6 +233,16 @@ Definition file, *usgcb-rhel5desktop-oval.xml-0.variables-0.xml* is the
 file containing exported variables from the XCCDF file, and
 *usgcb-results-oval.xml* is the the OVAL Result file.
 
+OVAL directives file can be used to control, whether results should be "thin" or "full".
```

Comment:
There is no need for the comma and I would suggest a preposition:
An OVAL directives file can be used to control whether results should be "thin" or "full".